common/lib/xmodule/xmodule/course_metadata_utils.py

```diff
@@ -0,0 +1,159 @@
+"""Simple utility functions that operate on course metadata.
+
+Ths is a place to put simple functions that operate on course metadata. It
+allows us to share code between the CourseDescriptor and CourseOverview
+classes, which both need these type of functions.
+"""
+from datetime import datetime
+from base64 import b32encode
+
+from django.utils.timezone import UTC
+
+from .fields import Date
+
+DEFAULT_START_DATE = datetime(2030, 1, 1, tzinfo=UTC())
+
+
+def clean_id(location, padding_char):
```

Comment:
Should this be clean_location? Or should the location variable be something else?